keyboards/chickenman/ciel65/readme.md

```diff
@@ -0,0 +1,17 @@
+# Ciel65
+
+A simple 65% keyboard based of the Bakeneko 65, originially made by Katano.
+
+* Keyboard Maintainer: [ChickenMan](https://github.com/chickenman112)
+* Hardware Supported: Ciel65 HS and Solder pcb
+* Hardware Availability: [To be opensourced](https://github.com/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make chickenman/ciel65/rev3:default
+
+Flashing example for this keyboard:
+
+    make chickenman/ciel65/rev3:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
Please add instructions to enter bootloader mode. (see [template readme](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md))

Reply:
Changed and added za stuffs

Let me know if there are any further issues